Week 2/Weekly Practice Problems (Week 2)/1688: Count of Matches in Tournament/agunn.java

```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public int numberOfMatches(int n) {
+        int count = 0;
+        while (n > 0) {
+            // Add matches to count
+            count = count + n/2;
+            // Advance teams
+            n = (n+1)/2;
+            if (n == 1) {
+                break;
+            }
```

Comment:
If we change our condition to `n > 1`, then this break actually becomes unnecessary! The next step after this is the loop condition, so it will automatically break for us.
```suggestion
```